css/reset.css

```diff
@@ -1,6 +1,43 @@
-/*  */
+/* reset style
+*/
```

Comment:
do you really need this comment?

Reply:
yes, I care a lot about this commit, but if it's annoying, I'll delete it))))))